actionpack/test/dispatch/cookies_test.rb

```diff
@@ -278,6 +278,11 @@ def noop
     def encrypted_cookie
       cookies.encrypted["foo"]
     end
+
+    def cookie_expires_in_two_hours
+      cookies[:user_name] = { value: "assain", expires: 2.hours }
+      head :ok
```

Comment:
@kaspth :smile: 
But, deleting the line throws:
`ActionController::UnknownFormat: CookiesTest::TestController#cookie_expires_in_two_hours is missing a template for this request format and variant.`